Snippets/Snippets_6/Callback/Usage.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Snippets6.Callback
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NServiceBus;
+
+    public class Usage
+    {
+        public async Task Describe()
+        {
+            IBus bus = null;
+            PlaceOrderResponse responseMessage;
+            var options = new SendOptions();
+
+            #region CallbackWithMessageAsResponse
+            responseMessage = await bus.RequestWithTransientlyHandledResponseAsync<PlaceOrderResponse>(new PlaceOrder(), options);
+            #endregion
+
+            #region TriggerCallbackWithMessageAsResponse
+            bus.Reply(new PlaceOrderResponse());
+            #endregion
+
+            #region CallbackWithEnumAsResponse
+            var response = await bus.RequestWithTransientlyHandledResponseAsync<LegacyEnumResponse<Status>>(new PlaceOrder(), options);
+            var responseEnum = response.Status;
+            #endregion
+
+            #region TriggerCallbackWithEnumAsResponse
+            bus.Reply(new LegacyEnumResponse<Status>(Status.OK));
+            #endregion
+
+            #region CancelCallback
+            var cs = new CancellationTokenSource();
+            options.RegisterCancellationToken(cs.Token);
+            try
+            {
+                responseMessage = await bus.RequestWithTransientlyHandledResponseAsync<PlaceOrderResponse>(new PlaceOrder(), options);
+            }
+            catch (OperationCanceledException ex)
```

Comment:
I was really on the fence how to express that an `OperationCancelledException` is raised. I decided to go with the empty catch. So maybe add a comment like

`// Exception that is raised when the CancellationTokenSource is canceled`

thoughts?